src/ui/widgets/PitServiceWidget/PitApproachRail/PitApproachRail.module.scss

```
@@ -74,153 +43,107 @@
-    left: 0;
-    right: 0;
-    height: 2px;
-    transform: translateY(1px);
-  }
+  transition: left $rail-glide linear;
 }
 
 // Sits on top of the lane, so it takes a shadow rather than a background: a
 // plate behind it would hide the very fill it is standing on.
-.inlineReadout {
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)